### ISSUE TITLE
`skupper network status` hangs and times out

### ISSUE THREAD

**Issue by waldner** [skupperproject/skupper#1293]
While other `skupper` commands work (although they take some time to produce output), `skupper network status` doesn't:

```
$ skupper network status -n myns 
Unable to retrieve network information: context deadline exceeded

Loading just local information:
The site configuration is not available: context deadline exceeded
```

Don't know where to look to debug this.

**Comment by nluaces** [comment]
Hi @waldner, this is a know issue that it's fixed in 1.5.X versions by a new implementation. Let us know what do you think.

**Comment by nluaces** [comment]
Related issues: #766 , #798
